auth0/resource_auth0_organization_test.go

```diff
@@ -137,20 +99,114 @@ resource auth0_organization acme {
+					resource.TestCheckResourceAttr("auth0_organization.acme", "connections.#", "2"),
+					resource.TestCheckResourceAttrSet("auth0_organization.acme", "connections.0.connection_id"),
+					resource.TestCheckResourceAttrSet("auth0_organization.acme", "connections.1.connection_id"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "connections.0.assign_membership_on_login", "false"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "connections.1.assign_membership_on_login", "true"),
+				),
+			},
+			{
+				Config: template.ParseTestName(testAccOrganizationUpdateAgain, strings.ToLower(t.Name())),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("auth0_organization.acme", "name", fmt.Sprintf("test-%s", strings.ToLower(t.Name()))),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "display_name", fmt.Sprintf("Acme Inc. %s", strings.ToLower(t.Name()))),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.#", "1"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.0.logo_url", "https://acme.com/logo.svg"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.0.colors.%", "2"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.0.colors.primary", "#e3e2f0"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.0.colors.page_background", "#e3e2ff"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "metadata.%", "0"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "connections.#", "1"),
+					resource.TestCheckResourceAttrSet("auth0_organization.acme", "connections.0.connection_id"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "connections.0.assign_membership_on_login", "false"),
+				),
+			},
+			{
+				Config: template.ParseTestName(testAccOrganizationUpdateAgainAndAgain, strings.ToLower(t.Name())),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("auth0_organization.acme", "name", fmt.Sprintf("test-%s", strings.ToLower(t.Name()))),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "display_name", fmt.Sprintf("Acme Inc. %s", strings.ToLower(t.Name()))),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.#", "1"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.0.logo_url", "https://acme.com/logo.svg"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.0.colors.%", "2"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.0.colors.primary", "#e3e2f0"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "branding.0.colors.page_background", "#e3e2ff"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "metadata.%", "0"),
+					resource.TestCheckResourceAttr("auth0_organization.acme", "connections.#", "1"),
```

Comment:
One thing I observed while working on the organization members is that for whatever reason, the bulk of this code had issues with completely clearing out members. It's possible that the connections has the same issue, so for that reason I think having the following test case would be useful:

```
resource.TestCheckResourceAttr("auth0_organization.acme", "connections.#", "0"),
```

Reply:
I added another test case where we remove all connections 👍🏻  and tests pass